lib/reporters/base.js

```diff
@@ -231,13 +235,16 @@ exports.list = function(failures) {
  * of tests passed / failed etc.
  *
  * @param {Runner} runner
+ * @param {Object} options runner optional options
  * @api public
  */
 
-function Base(runner) {
+function Base(runner, options) {
   var stats = this.stats = { suites: 0, tests: 0, passes: 0, pending: 0, failures: 0 };
   var failures = this.failures = [];
 
+  this.reporterOptions = options ? options.reporterOptions : null;
```

Comment:
I see. We could change this we decide to bump major though. Ideally, `outfile`/`output` would work for any reporter (that doesn't move the cursor around), and the reporter implementation could ignore it as long as it is using `Base#write`.

We could leave this improvement for later though.

/cc @boneskull